includes/class-freemius.php

```diff
@@ -736,6 +794,11 @@ function _submit_uninstall_reason_action() {
 
 			$this->_storage->store( 'uninstall_reason', $reason );
 
+			if ( ! $this->_plugin->is_plugin() ) {
+				// @todo Check slug/instance.
```

Comment:
@fajardoleo not sure what's the @todo means, is it still relevant?

In any case, please add some documentation that explains that with themes there's no uninstallation hook, and we need to trigger it on deactivation.

Also, I would add another helper function to the plugin entity called `is_theme()` and use it instead here. It makes the code clearer.